VALUES.md

```diff
@@ -0,0 +1,66 @@
+# Ayo.js Community Values
+
+Ayo.js is about the humans that help make it happen. It's a project aimed at
+creating a new foundation of project governance and management that brings
+**humans** and their concerns front and center.
+
+You can pronounce it however you want: be it “ey-yo”, “I-oh”, or “Awooooo” — the
+Ayo.js project strives to create an environment where you can feel at home and
+focus on your contributions to this nice shared space that we all benefit from.
+
+## Ayo.js is more than the code in its repository.
+
+Documentation, long-term support, translations and localizations,
+package management, libuv, V8 and Chakra, builds/ops, testing and its
+infrastructure, hardware and IoT, Android, streams, community events,
+facilitation, moderation and education. These and more are all concerns that
+can be handled separately and by members who might be much better qualified
+for these individual tasks than a central decision body.
+
+Ayo.js wants to go further than Node.js did in spreading responsibility across
```

Comment:
Can we standardize on "will" instead of "want" in this sentence, given that similar sentences all use "want"? I like how "will" makes a clear stand on what the project's goals are.